### PR TITLE
Minor change to ENS de Lyon style

### PR DESCRIPTION
**Pull request by smarchand**
I've changed the bibliography layout (indent and entry spacing)

### CODE REVIEW

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 1]
```diff
@@ -657,7 +663,7 @@
     </layout>
   </citation>
   <!--_-_-_-_-_-SECTION DEFINITION DE LA BIBLIO (bibliography)-_-_-_-_-_-_-->
-  <bibliography>
+  <bibliography  hanging-indent="true" entry-spacing="2">
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: are you sure about entry-spacing="2"? That's two (!) line breaks between entries. 

Entry 1
\
\
Entry 2

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 2]
```diff
@@ -19,7 +19,13 @@
     <category field="social_science"/>
     <category field="philosophy"/>
     <category field="literature"/>
-    <updated>2016-11-17T16:34:49+00:00</updated>
+    <category field="anthropology"/>
+    <category field="geography"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="sociology"/>
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: this seems a bit excessive in terms of disciplines. Wouldn't social_science and humanities just cover this? Too many categories make this less useful as metadata

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 3]
```diff
@@ -19,7 +18,11 @@
     <category field="social_science"/>
     <category field="philosophy"/>
     <category field="literature"/>
-    <updated>2016-11-17T16:34:49+00:00</updated>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="sociology"/>
```
Comment (rmzelle) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: @smarchand, this is still rather extreme, though. Could you please trim it back to 2-3 fields max? For comparison, of our 1435 independent CSL styles, just 5 have 4 fields assigned, and only one styles 5 fields. We currently have no styles with more assigned fields than that, while here you go all the way to 7.

In general, if a style can't be assigned to a select few fields, we assign it either no field at all or the "generic-base" field.

Comment (rmzelle) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: The CSL fields are a bit unevenly spread over the different disciplines anyway. Generally it's much easier to just search for styles by name than to filter them by discipline.

**Comment by rmzelle** [comment]
Great, thanks. Sorry for being nitpicky.

**Comment by smarchand** [comment]
thanks to you !

Stéphane

2017-01-25 14:45 GMT+01:00 Rintze M. Zelle <notifications@github.com>:

**Pull request merged**